### PR TITLE
Parameterize the client config

### PR DESCRIPTION
**Pull request by BartWillems**
I've parameterised the client config, now it's possible to choose the ownership & permissions of the config file.

### CODE REVIEW

File: manifests/client/config.pp [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # private class
-class mcollective::client::config {
+class mcollective::client::config (
+    $owner = 'root',
```
Comment (ekohl) [inline manifests/client/config.pp]: Maybe use the ```String``` datatype on all of these?

Comment (BartWillems) [reply manifests/client/config.pp]: Yes, of course

**Comment by bastelfreak** [comment]
@BartWillems thanks for this PR. Can you take a look at the used email address in your commit? It isn't associated with your github account.

**Comment by BartWillems** [comment]
@bastelfreak The email address in the commit is my work address, it is now added to my account; thanks for noticing 👍

**Pull request merged**